### PR TITLE
Release/snowplow web/0.12.3

### PR DESCRIPTION
**Pull request by emielver**
## Description & motivation
Quick fix for compiling issue

## Checklist
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)
- [x] I have added tests & descriptions to my models (and macros if applicable)
- [x] I have raised a [documentation](https://github.com/snowplow/documentation) PR if applicable (Link here if required)

## Release Only Checklist
- [x] I have updated the version number in all relevant places
- [x] I have updated the CHANGELOG.md

### CODE REVIEW

**Review by rlh1994 — APPROVED** [review]
Wait for the tests to pass but otherwise LGTM!

File: CHANGELOG [hunk 1]
```diff
@@ -1,10 +1,22 @@
+snowplow-web 0.12.3 (2022-12-28)
```
Comment (agnessnowplow) [inline CHANGELOG:1]:
```suggestion
snowplow-web 0.12.3 (2023-01-05)
```

Comment (rlh1994) [reply CHANGELOG]: will set to tomorrow as not going to release tonight, good catch!

Comment (agnessnowplow) [reply CHANGELOG]: otherwise good to go!

**Review by agnessnowplow — APPROVED** [review]
LGTM!

**Pull request merged**